README.md

```diff
@@ -79,11 +79,40 @@ Use the `LangChain` CLI to bootstrap a `LangServe` project quickly.
 
 To use the langchain CLI make sure that you have a recent version of `langchain-cli`
 installed. You can install it with `pip install -U langchain-cli`.
+## Setup
+**Note**: We use `poetry` for dependency management. Please follow poetry [doc](https://python-poetry.org/docs/) to learn more about it.
 
+### 1. Create new app using langchain cli command
 ```sh
-langchain app new ../path/to/directory
+langchain app new my-app
 ```
 
+### 2. Activate `poetry` shell. Every command after step 2 should be executed within `poetry` shell.
+```sh
+poetry shell
+```
+
+### 3. Now, install all the dependencies using `poetry`
+```sh
+poetry install
+```
+
+### 4. Install `langchain-openai` dependency to use ChatOpenAI llm. Do similar for other dependencies you plan to use.
```

Comment:
The first thing to do is to define the runnable in add_routes

so they need to go to server.py and edit `add_routes(app. NotImplemented)`

---

After that we can specify that they should add any 3rd party packages and set up env variables appropriately.

Reply:
Done.